codegen/templates/resource.go.tpl

```diff
@@ -61,7 +61,7 @@ func {{ .UpperCaseDifferentiator }}Resource() *schema.Resource {
 			{{- if .Schema.DisplayAttrs.Sensitive }}
 			Sensitive:   true,
 			{{- end }}
-			Description: "{{ .Description }}",
+			Description: `{{ .Description }}`,
```

Comment:
I changed the description to string literals because it turned out that some descriptions contain quotes, so if the description did as well, there were problems.